in_progress/roles.md

```diff
@@ -0,0 +1,109 @@
+# Abstract
+
+The purpose of this document is to formalize the governance process used by the `scverse` project, to clarify how decisions are made and how the various elements of our community interact.
+
+As a whole, we are a consensus-based community project.
+However, any subproject within the scverse can have their own governance system as long as they do not conflict with the rules set out here.
+Anyone with an interest in the project can join the community, contribute to the project design, and participate in the decision making process.
+This document describes how that participation takes place, how to find consensus, and how deadlocks are resolved.
+
+# Roles And Responsibilities
+
+## The Community
+
+The scverse community consists of anyone using or working with the project in any way.
+
+## Contributors/ Project developers
+
+A community member can become a contributor by interacting directly with the project in concrete ways, such as:
+
+- proposing, discussing, or reviewing a change to the code, documentation, or specification via a GitHub pull request to any https://github.com/scverse repository;
+- reporting a GitHub issue to any https://github.com/scverse repository;
+- discussing examples or usage issues on any of our discussion platforms
+
+among other possibilities.
+Any community member can become a contributor, and all are encouraged to do so.
+By contributing to the project, community members can directly help to shape its future.
+
+All community members are required to adhere by our code of conduct (**TODO: add link**).
+Potential contributors are encouraged to read the Contributing Guide (**TODO: add link**).
+
+A community member becomes a contributor when the following criteria are met:
+
+- At least two core developers from separate employers agree membership is a good idea.
+- The new member has supported the project several times, either through code or otherwise
+
+## Core team
```

Comment:
To simplify the roles I would merge the core team and core developers.
They are making contributions in different ways (e.g. outreach vs project maintenance), but both are significant. No need to distinguish them in my opinion.

Reply:
> No need to distinguish them in my opinion.

I think we do need to distinguish for things like responsibilities. E.g. outreach members probably don't need to know a ton about the review process, or to have commit rights to all projects. Also probably don't need to go to technical meetings. I think this keeps the roles themselves simple.

I would agree that for many purposes we otherwise don't need to distinguish.

Reply:
True, but it is also not like it would hurt if they had commit rights. Remember, in nf-core **every** member has commit access to **all** repositories (very few exceptions) and people are surprisingly reluctant to merge things that they should not be doing. This happens like almost never. An outreach member of scverse would know how to behave and this would never happen, I am sure. We are talking about like 5 people or something here.

I don't mind having two roles, but the more complex we make this structure, the more work we will always have. Twice the listings of people, twice the role descriptions, [...]

Reply:
We could give everyone on the core team commit rights, though I also remember nf-core saying they were re-thinking the commit rights thing. I still think documenting specific roles would still be valuable.

A very common piece of advice on project management I've been getting is to have clear documented roles. I think this becomes very important for "who's job is it to moderate forums", "who responds to conduct complaints", "who could merge a critical bug fix to this repo". One of the key features noted here was it makes it explicit what roles are currently empty and who you need to recruit/ hire.

Another is letting people know what they're signing up for. If we say "core members are responsible for the twitter account, managing the community, and reviewing PRs", that's a lot of different things!

Reply:
> We could give everyone on the core team commit rights, though I also remember nf-core saying they were re-thinking the commit rights thing. I still think documenting specific roles would still be valuable.
> 
> A very common piece of advice on project management I've been getting is to have clear documented roles. I think this becomes very important for "who's job is it to moderate forums", "who responds to conduct complaints", "who could merge a critical bug fix to this repo". One of the key features noted here was it makes it explicit what roles are currently empty and who you need to recruit/ hire.
> 
> Another is letting people know what they're signing up for. If we say "core members are responsible for the twitter account, managing the community, and reviewing PRs", that's a lot of different things!

I can totally understand your position. Think we should discuss this tomorrow :)